src/components/structures/RightPanel.js

```diff
@@ -190,7 +204,7 @@ export default class RightPanel extends React.Component {
 
         return (
             <aside className={classes}>
-                { panel }
+                {panel}
```

Comment:
I think this change is unintended as the code-style at least used to prefer spaced when children

Reply:
This not being documented in the code-style but simply following the style of the rest of the code base (consistency is most important)